man1/aur-fetch-snapshot.1

```diff
@@ -0,0 +1,32 @@
+.TH AUR-FETCH 1 2018-02-14 AURUTILS
+.SH NAME
+aur-fetch-snapshot \- fetches a tar archive from a given URL.
+
+.SH SYNOPSIS
+.SY aur
+.B fetch
+.B snapshot
+.OP -L log_dir 
+.YS
+
+.SH DESCRIPTION
+\fIaur-fetch-snapshot\fR accepts URLs from stdin, and attempts to
+download and extract tar archives to a given location.
+
+.SH OPTIONS
+.B \-L
+.RS
+The location of where the package should be downloaded.
```

Comment:
The location where build files are downloaded to is `$PWD`. `-L` is the location where _diffs_ (from previous unpacked archives) are stored. (example: https://github.com/AladW/aurutils/blob/master/lib/aur-sync#L264)